assets/js/googlesitekit/datastore/user/surveys.js

```diff
@@ -30,24 +30,30 @@ import {
 	commonActions,
 	combineStores,
 	createRegistrySelector,
+	createReducer,
 } from 'googlesitekit-data';
 import { CORE_USER, GLOBAL_SURVEYS_TIMEOUT_SLUG } from './constants';
 import { createFetchStore } from '../../data/create-fetch-store';
 import { createValidatedAction } from '../../data/utils';
 
 const fetchTriggerSurveyStore = createFetchStore( {
 	baseName: 'triggerSurvey',
-	controlCallback: ( { triggerID } ) => {
-		return API.set( 'core', 'user', 'survey-trigger', { triggerID } );
+	controlCallback: ( { triggerID, ttl } ) => {
+		const args = ttl ? { ttl } : {};
+		return API.set( 'core', 'user', 'survey-trigger', {
+			triggerID,
+			...args,
```

Comment:
I originally added the `ttl` unconditionally but it would have required many tests to be updated as there were several the have `toHaveFetched` with a specific body. Rather than modifying the tests to assert a partial (as some have been here), we pass it if it's a non-falsy value. It defaults to `0` and has no effect if it's not a positive value, so this should be safe.